rllib/examples/envs/classes/multi_agent/footsies/test/test_footsies.py

```diff
@@ -0,0 +1,132 @@
+import os
+import platform
+import sys
+import tempfile
+import time
+import unittest
+from contextlib import contextmanager
+from pathlib import Path
+
+from ray.rllib.env import EnvContext
+from ray.rllib.examples.envs.classes.multi_agent.footsies.footsies_env import (
+    env_creator,
+)
+
+# Detect platform and choose appropriate binary
+if platform.system() == "Darwin":
+    binary_to_download = "mac_headless"
+elif platform.system() == "Linux":
+    binary_to_download = "linux_server"
+else:
+    raise RuntimeError(f"Unsupported platform: {platform.system()}")
```

Comment:
This is great!

Can we add it to the main examples and hence remove arg that requires user to specify OS manually?

Reply:
Yep, thanks. Done!